GoogleTestAdapter/Core/Resources.resx

```diff
@@ -0,0 +1,444 @@
+  </data>
+  <data name="ReportedResultsToVS" xml:space="preserve">
+    <value>{0} Reported {1} test results to Visual Studio during test execution, executable: '{2}'</value>
+    <comment>{0} represents an unlocalized name for thread (e.g. "T001519"), {1} represents a number (e.g. 5), {2} represents a unlocalized name of executable (e.g. cmd.exe) file</comment>
+  </data>
+  <data name="ReportedTestResults" xml:space="preserve">
+    <value>{0} Reported {1} test results to Visual Studio, executable: '{2}', duration: {3}</value>
+    <comment>{0} represents an unlocalized name for thread (e.g. "T001519"), {1} represents a number (e.g. 5),{2} represents an unlocalized name of executable (e.g. cmd.exe) file, {3} means stopwatch time</comment>
+  </data>
+  <data name="Results" xml:space="preserve">
+    <value>Test {0}: Expected {1} ms, actual {2} ms</value>
+    <comment>{0} represents an unlocalized name for thread (e.g. "T001519"), {1} represents a number (e.g. 5), {2} represents a number (e.g. 5)</comment>
+  </data>
+  <data name="RunBatchException" xml:space="preserve">
+    <value>{0} {1} batch file caused exception, message: '{2}', executed command: '{3}'</value>
+    <comment>{0} represents an unlocalized name for thread (e.g. "T001519"), {1} represents a localized name for test mode (e.g. "Test setup" or "Test teardown") , {2} represents localized exception message (e.g. "Cannot load file"), {3} represents unlocalized file path (e.g c:\test\foo.bat)</comment>
+  </data>
+  <data name="RunExecutableError" xml:space="preserve">
+    <value>{0} Failed to run test executable '{1}': {2}</value>
+    <comment>{0} represents an unlocalized name for thread (e.g. "T001519"), {1} represents a unlocalized name of executable (e.g. cmd.exe) file, {2} represents localized exception message (e.g. "Cannot load file")</comment>
+  </data>
+  <data name="SchedulingStats" xml:space="preserve">
+    <value>&gt;&gt;&gt; Scheduling statistics &lt;&lt;&lt;</value>
+  </data>
+  <data name="Settings" xml:space="preserve">
+    <value>Settings for test executable '{0}': {1}</value>
+    <comment>{0} represents a unlocalized name of executable (e.g. cmd.exe) file, {1} means current object</comment>
+  </data>
+  <data name="SettingsMessage" xml:space="preserve">
+    <value>Settings for test executable '{0}': {1}</value>
+    <comment>{0} represents a unlocalized name of executable (e.g. cmd.exe) file, {1} represents an unlocalized settings string</comment>
+  </data>
+  <data name="SettingsWrapperString" xml:space="preserve">
+    <value>SettingsWrapper is already running with settings for an executable on thread '{0}', can not also be used by thread {1}</value>
+    <comment>{0} represents an unlocalized name for thread (e.g. "T001519"), {1} represents an unlocalized name for thread (e.g. "T001519")</comment>
+  </data>
+  <data name="StackTrace" xml:space="preserve">
+    <value>{0} Stack trace: {1}{2}</value>
```

Comment:
{1} (new line) is explicitly "\n" in many other strings here. Please make this consistent.